stack.yaml

```diff
@@ -1,26 +1,19 @@
-resolver: lts-13.26
+resolver: lts-17.6
 # Resolves hjsmin requiring language-javascript ==0.6, while the purescript
 # compiler requires language-javascript ==0.7.
 allow-newer: true 
 packages:
 - '.'
 extra-deps:
+- barrier-0.1.1
 - bytestring-trie-0.2.5.0
-- cborg-0.2.2.0
-- connection-0.3.1
-- happy-1.19.9
+- classy-prelude-yesod-1.5.0
 - language-javascript-0.7.0.0
-- network-3.0.1.1
-- serialise-0.2.2.0
-- socks-0.6.0
-- these-1.0.1
-- semialign-1
-# purescript 0.14.0-rc4
+# purescript 0.14.3
 - github: purescript/purescript
-  commit: 8663d00a27c2c36e229f3d2f98358e660c297335
+  commit: b4c90a8bb6a1059f271f4211687192370fda81bc
```

Comment:
I think we don't need to refer to the repo on github any more; I think this ought to become `purescript-0.14.3` once it's on Hackage so that we can be sure we're referring to a released version.

Reply:
I can let this sit until language-cst makes it onto Hackage.